src/Apps/Juno/Command.hs

```diff
@@ -38,80 +43,85 @@ setBalances :: Map.Map Text Rational -> DEval.PersistentState -> DEval.Persisten
 setBalances bals ps = ps { DEval._persistentBalances = bals }
 
 --JunoEnv:  MVar (State, Map(TX -> Swift))
-runCommand :: JunoEnv -> CommandEntry -> IO CommandResult
-runCommand env cmd' = do
+runCommand :: JunoEnv -> (CommandEntry, RequestId) -> IO CommandResult
+runCommand env (cmd', rid) = do
   mvar <- return $ getStateMVar env
-  (ps, ss) <- MV.takeMVar mvar -- persistent s, swift
+  (ps, ss, store) <- MV.takeMVar mvar -- persistent s, swift, prog input store
   let bals = balances ps
   case readHopper $ unCommandEntry cmd' of
     Left err -> do
-      MV.putMVar mvar (ps,ss)
+      MV.putMVar mvar (ps, ss, store)
```

Comment:
... so that here we can just say `org` instead of this. better yet, add a `let` for a `noop` function that does this, and use everywhere we're copypasting this noop.